js/os.ts

```diff
@@ -1,11 +1,18 @@
 // Copyright 2018 the Deno authors. All rights reserved. MIT license.
-import { ModuleInfo } from "./types";
 import * as msg from "gen/msg_generated";
 import { assert } from "./util";
 import * as util from "./util";
 import * as flatbuffers from "./flatbuffers";
 import { sendSync } from "./dispatch";
 
+interface ModuleInfo {
```

Comment:
No longer @internal?

Why is this moved out of types? I guess because it relies on msg.MediaType ?

Maybe this should be in compiler.ts ?

Reply:
Well it isn't exported anymore so it shouldn't leak.  It was only `@internal` because it was an explicit export of a module, but I forgot to double check that somehow it didn't leak.

I will see if it fits better in `compiler.ts`.